api/openapi-spec/swagger.json

```diff
@@ -4538,6 +4538,9 @@
        "$ref": "#/definitions/v1.Port"
       }
      },
+     "proxy": {
```

Comment:
Please consider to call is masquerade or nat.

To me at least, proxy is rather a process which does the relaying.
And I'd also like to keep the process bound proxying free, because I hope that we'll have some process bound proxying in future, as it's the only K8s compatible way to do things.

Reply:
Why would the App Operator care if the proxy is implemented by a process, or as suggested here, by the kernel? I would assume that if we find an efficient way to do what you hope, we can just flip the implementation.

Why is kernel-based less k8s compatible?

(I'm not going for a holy war about the name `proxy`, just willing to understand your position)

Reply:
Please reach out to @SchSeba to understand the differences between a kernel and process based proxy approach for kubernetes.

A user would usually not care, but if sees a spec, then I would like to leave the best hints to give a user transparency. That#s why I favor to have a more kernel-ish name for this implementation and `proxy` for a more process-ish implementation.